packages/features/ee/workflows/components/scrollable.css

```diff
@@ -0,0 +1,3 @@
+.scrollable-content {
```

Comment:
We should have implemented this fix using Tailwind's utility classes instead of traditional CSS.